### ISSUE TITLE
Followup documentation for petroff color sequence

### ISSUE THREAD

**Issue by tacaswell** [matplotlib/matplotlib#28750]
This can be done in a followup, but would be nice if this got added to the following for discoverability: 

 ~* https://matplotlib.org/devdocs/users/explain/colors/colormaps.html~
~* https://matplotlib.org/devdocs/gallery/color/colormap_reference.html#sphx-glr-gallery-color-colormap-reference-py~
- [x] add new example to https://matplotlib.org/devdocs/gallery/style_sheets/index.html showing how to use the color-sequences #28796
- [ ] add a page explaining the color sequences to https://matplotlib.org/devdocs/users/explain/colors/index.html 

ETA: also I'm really psyched this is going in, thanks for your work on this folks!

_Originally posted by @story645 in https://github.com/matplotlib/matplotlib/pull/27851#pullrequestreview-2244421580_

**Comment by tacaswell** [comment]
But we added these as color sequences not as categorical color maps.

**Comment by tacaswell** [comment]
@story645 Can you either edit the initial post or leave a comment with details about what you have in mind?

**Comment by tacaswell** [comment]
I think we made all of the discrete color maps available as sequences, but did not remove them as color maps.  With `petroff10` we did not also add it as a colormap (maybe someone will ask for that, but I think we should hold off on doing it until there is actually a demand).

**Comment by Harish-2003** [comment]
May I know what is '''petroff10'''

**Comment by Nikhil172913832** [comment]
Hey, is the issue still open, would love to contribute?

**Comment by matthewfeickert** [comment]
Note that issues don't get assigned or claimed: https://matplotlib.org/devdocs/devel/contribute.html#work-on-an-issue

https://github.com/matplotlib/matplotlib/blob/5461ae29459a7cef8342289782ca0cbe59fd7545/doc/devel/contribute.rst#L246-L251

So go for it. :+1:

**Comment by ammar-qazi** [comment]
I was trying to set up a dev environment for matplotlib. Finally, figured it out: had to switch from Windows to Linux. So, I'll working on this issue now.

**Comment by hwii-florescent** [comment]
Is the issue still opened? I want to contribute to it

**Comment by ammar-qazi** [comment]
@hwii-florescent Yeah, I worked on the first task. You can get started on the second task:

>  add a page explaining the color sequences to https://matplotlib.org/devdocs/users/explain/colors/index.html

**Comment by Nikhil172913832** [comment]
I am willing to take on the second task. Could you provide more context on what exactly needs to be done?

**Comment by ammar-qazi** [comment]
Here's my understanding. 

The page the second task links to has this in its description 

> Colors

> Matplotlib has support for visualizing information with a wide array of colors and colormaps. These tutorials cover the basics of how these colormaps look, how you can create your own, and how you can customize colormaps for your use case.
> 
> For even more information see the [examples page](https://matplotlib.org/devdocs/gallery/color/index.html#color-examples).

As of now, the page doesn't explore the color sequences — as in how you can customize the color maps and make your information more accessible by using different style sheets.

So, if we have a separate page for that, it would be helpful to Matplotlib users. 

That's all I know.

**Comment by u7228810** [comment]
Hello, is this issue completed yet? Am I able to work on it as my first issue - maybe finish off what others left if its still open?

**Comment by ammar-qazi** [comment]
@u7228810 Yes, the second task hasn't been attempted yet. 

That said, if it's your very first contribution, I would suggest starting with a difficulty: easy task.

**Comment by scottshambaugh** [comment]
> we added these as color sequences not as categorical color maps

FWIW I'm clueless on the difference, I thought that the listings on this page were exhaustive! https://matplotlib.org/devdocs/users/explain/colors/colormaps.html

Unclear to me how to use this new coloring without changing the whole style sheet.

**Comment by timhoffm** [comment]
We have both

- colormaps in the ColorMapRegistry `mpl.colormaps` / `plt.colormaps`: mapping a numeric interval [0, 1] to colors.
- color sequences in the ColorSequenceRegistry `mpl.color_sequences`: named lists of colors 

Example usages:
```
ax. set_prop_cycle(color=plt.color_sequences['petroff10'])
```
```
ax.stackplot([0, 1], [[0, 1], [0, 2], [0, 3]], colors=plt.color_sequences['petroff10'])
```

**Comment by SHRISHTISHUKLA-0** [comment]
@ammar-quzi  Hello does Second task still open ?? pls let me know.

**Comment by ammar-qazi** [comment]
@SHRISHTISHUKLA-0 Yes, the second task is still open.

**Comment by SHRISHTISHUKLA-0** [comment]
Okay I want to work on this
Thank you)

**Comment by SHRISHTISHUKLA-0** [comment]
Can you please help to know about second issue deeply it will help to
understand what exactly and  where I have to work!!

**Comment by SHRISHTISHUKLA-0** [comment]
> > "add a page explaining the color sequences to https://matplotlib.org/devdocs/users/explain/colors/index.html"
> 
> [Color sequences](https://matplotlib.org/devdocs/api/_as_gen/matplotlib.colors.ColorSequenceRegistry.html) are named lists of colors that can be used for style sheets, [cycler objects](https://matplotlib.org/devdocs/users/explain/artists/color_cycle.html) and as [lists of colors](https://matplotlib.org/devdocs/users/explain/artists/color_cycle.html). [colors/index](https://matplotlib.org/devdocs/users/explain/colors/index.html) is the section of the docs that explains colors and it's missing a page on sequences.
> 
> If you've never worked with color sequences or don't see a need to, then you may not have the context necessary for writing this documentation.

hello I think that I can able to solve this is but here I can't able to link PR why?

**Comment by WeatherGod** [comment]
The error message indicates that you are trying to push to matplotlib's
repository rather than your fork. Make sure you follow the developer
guidance to help you get started:
https://matplotlib.org/devdocs/devel/development_workflow.html

**Comment by SHRISHTISHUKLA-0** [comment]
> The error message indicates that you are trying to push to matplotlib's
> repository rather than your fork. Make sure you follow the developer
> guidance to help you get started:
> https://matplotlib.org/devdocs/devel/development_workflow.html
> […](#)

ooh thank you as firstyear student first time I'm contributing in open sources and pr that's why I having lot of doubts.

**Comment by rcomer** [comment]
I think the remaining work to add a User Guide section is orthogonal to the petroff10 work, so better to continue that discussion at #30071.
